sktime/transformations/series/summarize.py

```diff
@@ -750,3 +750,140 @@ def get_test_params(cls, parameter_set="default"):
         params4 = {"summary_function": None, "quantiles": (0.1, 0.2, 0.25)}
 
         return [params1, params2, params3, params4]
+
+
+class SplitterSummarizer(BaseTransformer):
+    """Create summary values of a time series' splits.
+
+    A series-to-series transformer that applies a series-to-primitives transformer
+    to each series' train split.
+
+    It splits the series using a specified splitter and transforms each resulting
+    split using the provided transformer.
+
+    This transformer aims to provide a summarization of the series based on the
+    transformer and splitter given.
+
+    Attributes
+    ----------
+    transformer : series-to-primitives transformer used to convert series
```

Comment:
in the docstring style we use, the first row would precisely specify the type (`sktime` transformer inheriting from `BaseTransformer`), the second and subsequent rows explain what it is and how it impacts the outputs of the methpds.

Reply:
I guess it is fixed now